test/fixtures/apps/ts/tsconfig.json

```
@@ -0,0 +1,15 @@
+{
+    "compilerOptions": {
+      "target": "es2017",
+      "module": "commonjs",
+      "baseUrl": ".",
+      "strict": true,
+      "lib": [
+        "es2017"
+      ],
+      "skipLibCheck": true
```

`skipLibCheck` should not set to `true`, and it will ignore type check for `index.d.ts`

```suggestion
      "skipLibCheck": false
```